evm.md

```diff
@@ -2070,12 +2070,12 @@ There are several helpers for calculating gas (most of them also specified in th
     rule <k> Cselfdestruct(SCHED, ISEMPTY:Bool, BAL)
           => Gselfdestruct < SCHED > +Int Cnew(SCHED, ISEMPTY andBool Gselfdestructnewaccount << SCHED >>, BAL) ... </k>
 
-    syntax Int ::= Cgascap ( Schedule , Int , Int , Int ) [function]
-                 | Csstore ( Schedule , Int , Int , Int ) [function]
-                 | Rsstore ( Schedule , Int , Int , Int ) [function]
-                 | Cextra  ( Schedule , Bool , Int )      [function]
-                 | Cnew    ( Schedule , Bool , Int )      [function]
-                 | Cxfer   ( Schedule , Int )             [function]
+    syntax Int ::= Cgascap ( Schedule , Int , Int , Int ) [function, functional]
```

Comment:
This uses `#allBut64th` which is not marked `functional`, please either mark dependencies as `functional` or don't mark this one as functional.

Reply:
done